app/components/maps/MapOfLocations.jsx

```diff
@@ -61,7 +61,7 @@ class MapOfLocations extends React.Component {
                 <AccordionItem
                   key={loc.address.id}
                   title={loc.address.address_1}
-                  headerRenderer={title => (
+                  headerRenderer={(
```

Comment:
Since you got rid of the kind of weird title argument to the `headerRenderer` render prop, maybe you could also just get rid of the `title` prop at all? I see no reason to have two props do the same thing, except one is allowed to be an arbitrary `JSX.Element` while the other is restricted to being a string.